sdk/core/Azure.Core/tests/ManagementRecordedTestBaseTests.cs

```diff
@@ -11,6 +11,7 @@
 namespace Azure.Core.Tests.Management
 {
     [Parallelizable]
+    [PlaybackOnly("These are fake clients that won't run live")]
```

Comment:
How do you re-record these if tests fail when running Live?

Reply:
these are fake clients they don't need to be recorded either.